docs/vectorizer.md

```diff
@@ -69,6 +69,13 @@ Timescale Cloud or on a self-hosted Postgres server.
   vectorizer
   ```
 
+To validate that your API key is set correctly, run the following SQL query:
```

Comment:
This is true on self-hosted but not on cloud yet. Unfortunately. Eventually that should be true. Don't know  if we want to move this to self-hosted section by now?